config/server.py

```diff
@@ -0,0 +1,196 @@
+import uuid
+import argparse
+from urllib.parse import urlparse
+from functools import wraps
+
+import connexion
+import flask
+from google.oauth2 import id_token
+from google.auth.transport import requests
+from peewee import CharField, Model, SqliteDatabase
+
+
+def get_route_model(db):
```

Comment:
This route (type) factory doesn't really work as there is no superclass for the type it is creating, hence it's not possible to say what type of object is returned. Your data mapper, which implicity knows what the return type looks like, is coupled to the peewee ORM model. This likely does not decouple the data mapper from the underlying database (assuming peewee only deals with SQL and is not general purpose).